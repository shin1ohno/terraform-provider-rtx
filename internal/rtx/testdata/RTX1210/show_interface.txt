Interface LAN1
  Status: up
  IPv4: 192.168.1.254/24
  IPv6: Not configured
  Ethernet address: 00:A0:DE:AB:CD:01
  MTU: 1500

Interface LAN2
  Status: down
  IPv4: Not configured
  IPv6: Not configured
  Ethernet address: 00:A0:DE:AB:CD:02
  MTU: 1500

Interface WAN1
  Status: up
  IPv4: 203.0.113.1/30
  IPv6: 2001:db8::1/64
  Ethernet address: 00:A0:DE:AB:CD:03
  MTU: 1500

Interface PP1
  Status: up
  Type: PPPoE
  MTU: 1454

Interface VLAN10
  Status: up
  IPv4: 10.10.10.1/24
  Ethernet address: 00:A0:DE:AB:CD:04
  MTU: 1500