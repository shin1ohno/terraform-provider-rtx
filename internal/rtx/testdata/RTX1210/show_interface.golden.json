[
  {
    "name": "LAN1",
    "kind": "lan",
    "admin_up": true,
    "link_up": true,
    "mac": "00:A0:DE:AB:CD:01",
    "ipv4": "192.168.1.254/24",
    "mtu": 1500
  },
  {
    "name": "LAN2",
    "kind": "lan",
    "admin_up": false,
    "link_up": false,
    "mac": "00:A0:DE:AB:CD:02",
    "mtu": 1500
  },
  {
    "name": "WAN1",
    "kind": "wan",
    "admin_up": true,
    "link_up": true,
    "mac": "00:A0:DE:AB:CD:03",
    "ipv4": "203.0.113.1/30",
    "ipv6": "2001:db8::1/64",
    "mtu": 1500
  },
  {
    "name": "PP1",
    "kind": "pp",
    "admin_up": true,
    "link_up": true,
    "mtu": 1454
  },
  {
    "name": "VLAN10",
    "kind": "vlan",
    "admin_up": true,
    "link_up": true,
    "mac": "00:A0:DE:AB:CD:04",
    "ipv4": "10.10.10.1/24",
    "mtu": 1500
  }
]